core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml

```diff
@@ -1,48 +1,42 @@
 ---
-- import_tasks: gather-facts.yml
+
+- name: Check system index file existance
+  uri:
+    url: http://localhost/epirepo/helm-charts/system/index.yaml
+    method: HEAD
+  register: uri_index_test
+      # fail with error if status is unexpected
+  failed_when: uri_index_test.status is undefined or uri_index_test.status <= 0
```

Comment:
According to https://docs.ansible.com/ansible/2.8/modules/uri_module.html#return-values `status` is always returned.
I don't think there can be status code lower than 100 (https://en.wikipedia.org/wiki/List_of_HTTP_status_codes).

How about:
`failed_when: uri_index_test.status not in [200, 404]`

Reply:
as for now we do not have any system charts